RecoEgamma/ElectronIdentification/plugins/ElectronHEEPIDValueMapProducer.cc

```diff
@@ -130,6 +131,13 @@ class ElectronHEEPIDValueMapProducer : public edm::stream::EDProducer<> {
     return handles;
   }
   
+  template<typename T> bool isEventAOD(const edm::Event& iEvent,const DualToken<T>& token){
```

Comment:
this can be a const method ```bool isEventAOD(...) const ```

Reply:
thanks for the catch! I made it static instead as it doesnt really rely on any values of the object. I also corrected the above two functions.